src/index.ts

```diff
@@ -90,15 +90,10 @@ export {
   FromSourceOptions,
   FromManifestOptions,
 } from './collections';
-export {
-  RegistryAccess,
-  registry,
-  getCurrentApiVersion,
-  MetadataRegistry,
-  MetadataType,
-  DecompositionStrategy,
-  RecompositionStrategy,
-  TransformerStrategy,
-} from './registry';
+
+export { RegistryAccess, registry, getCurrentApiVersion, MetadataRegistry, MetadataType } from './registry';
+
+// TODO: don't export these strategies
```

Comment:
Is there a work item for this?

Reply:
No.  It's more of a thing I look around for when I do a major of a library.